packages/link/__stories__/TextLink.stories.js

```diff
@@ -1,20 +1,30 @@
 import React from 'react';
+import Link from '../lib/Link';
 import TextLink from '../lib/TextLink';
 
 export default {
-  argTypes: {onClick: {action: 'onClick'}},
+  component: TextLink,
+  subcomponents: {Link},
```

Comment:
this is handy for components like this that are closely related to other components because it adds the props documentation for subcomponents to the props table, which looks like this:
<img width="1036" alt="Screen Shot 2022-02-08 at 12 52 49 PM" src="https://user-images.githubusercontent.com/9812299/153073827-26f51314-0516-4e54-9e41-99444d83ae44.png">